### PR TITLE
Luw 43 Refactor Footer

### PR DESCRIPTION
**Pull request by fannymartensson**
- Refactored footer.
- Refactored navbar width.
- Refactored lineheight on theme typography and fixed styling on callToAction section.
- 
Screenshot 2022-10-05 at 09 52 13

### CODE REVIEW

**Comment by fannymartensson** [comment]
- Refactored footer.
- Refactored navbar width.
- Refactored lineheight on theme typography and fixed styling on callToAction section.

File: src/components/Sections/Footer/Footer.tsx [hunk 1]
```diff
@@ -1,100 +1,91 @@
...
 
-                <Link underline={'always'} href={openEmail}>
-                  {data.emailAddress}
-                </Link>
+  return (
+    <Grid className={classes.root}>
+      <Container className={classes.container}>
+        <Box component={'div'} className={classes.flexBox}>
+          {data.headline.map((column: headline, index: number) => (
+            <Box key={index} component={'div'} className={classes.textBox}>
```
Comment (olivernygren) [inline src/components/Sections/Footer/Footer.tsx:21]: `index` should never really be used as a `key`, can we use something else here for `key`?

File: src/components/Sections/Footer/Footer.tsx [hunk 2]
```diff
@@ -1,100 +1,91 @@
...
               </Typography>
+              {column.subtitle && (
+                <Typography className={classes.text} variant={'body1'}>
+                  {column.subtitle}
+                </Typography>
+              )}
+              {column.socialLinks &&
+                column.socialLinks.map((link: socialLinks, index: number) => (
+                  <Box
+                    key={index}
```
Comment (olivernygren) [inline src/components/Sections/Footer/Footer.tsx:33]: `index` should never really be used as a `key`, can we use something else here for `key`?

File: src/components/Sections/Footer/Footer.tsx (hunk 1 above)
Comment (fannymartensson) [reply src/components/Sections/Footer/Footer.tsx:21]: Yes will change it to id instead

File: src/components/Sections/Footer/Footer.tsx (hunk 2 above)
Comment (fannymartensson) [reply src/components/Sections/Footer/Footer.tsx:33]: Yes will change it to id instead

**Pull request merged**